pkgs/applications/networking/cluster/atmos/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "atmos";
+  version = "1.4.28";
+
+  src = fetchFromGitHub {
+    owner = "cloudposse";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-AqW//+x1tKzm5dJirXCLFuDBTk3dHHw7c6vwsVsI37U";
+  };
+
+  vendorSha256 = "sha256-bepv8Ns8vcpLpmO7fsRgGijB9B6kQ5HhxUhIuEpGtOY=";
+
+  ldflags = [ "-X 'github.com/cloudposse/atmos/cmd.Version=${version}'" ];
+
+  doCheck = false;
+
+  doInstallCheck = true;
+  installCheckPhase = ''
```

Comment:
Is it possible to use a more standardized way to check this? Something like
``` 
 passthru.tests.version = testers.testVersion {
   package = opensupaplex;
   command = "opensupaplex --help";
   version = "v${version}";
 };
```

Reply:
I do it via `installCheckPhase` much more often and there's been a lot of discussions around the positives and negatives (I can't find any links ATM)

It's not really standardized, just completely random

```
λ rg "version \| " --files-with-matches | grep ".nix" | grep -v "/tests/" | wc -l
73
λ rg "testers\.testVersion \{" --files-with-matches | wc -l
131
```

my main reasons for not doing the passthru is 1: I need to remember to build `xyz.passthru.tests.version`, 2: I don't like having to import the package into itself to provide it to the test passthru because it's hideous

Reply:
Ok, I will leave this as-is then.